lms/djangoapps/courseware/module_render.py

```diff
@@ -182,6 +183,7 @@ def toc_for_course(request, course, active_chapter, active_section, field_data_c
                                      })
             toc_chapters.append({
                 'display_name': chapter.display_name_with_default,
+                'display_id': name_without_spaces,
```

Comment:
@andy-armstrong This isn't a bug. Some accessibility work that was done, sends focus to the first item when the menu expands. So what you're seeing is the first item with focus (I imagine you just opened the menu) and the second which is active.